next.config.js

```diff
@@ -29,6 +29,10 @@ module.exports = (phase) =>
           });
           return config;
         },
+        i18n: {
```

Comment:
This appears to only be supported since [Next.js v10.0.0](https://nextjs.org/docs/advanced-features/i18n-routing#:~:text=Next.js%20has%20built%2Din%20support%20for%20internationalized%20(i18n)%20routing%20since%20v10.0.0). Are we sure this is generating the lang attribute to the <html> tag on v9.3.2?